### DISCUSSION TITLE
烂尾楼-地理-人口

### DISCUSSION THREAD

**Discussion by suncunhu** [WeNeedHome/SummaryOfLoanSuspension#1091] [Ideas]
观察全国停贷地图
发现从北往南的各个省份，排除直辖市
从山东-江苏-浙江-福建-广东的第一线，
从辽宁-河北-河南-湖北-湖南-广西的第二线，
从山西-陕西-四川-云南的第三线。

第一线沿海省份经济发展的早，烂尾楼虽然也有，但数量不及第二线和第三线省份，尤其是第二线省份，烂尾楼最为严重。
出现如今遍地烂尾楼的情况是必然的，也别太指望会有人来解决。

**Comment by suncunhu** [comment]
普通没啥钱的年轻人的婚恋面对的也是这个情况

没钱泡不到白富美(一线城市新房)
只能退而求其次，钓个黑木耳，
结果彩礼投进去了，发现黑木耳被玩烂生不了孩子(二线烂尾楼)

**Reply by ghost** [reply]
那为什么不找婊子？

**Reply by suncunhu** [reply]
@Hashiramaff  嫖类似于租房，没法律保护，长期来看风险很高。

**Reply by ghost** [reply]
违背妇女意志警告⚠️，天朝不愧为第一护B大国🤣。
